bin/e2e/delete-profile-user.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpStatus } from "@fwl/web";
+
+import {
+  ConnectProfileAdminApi,
+  Configuration,
+} from "../../connect-profile-client";
+
+const adminClient = new ConnectProfileAdminApi(
+  new Configuration({
+    apiKey: `API_KEY ${process.env.PROFILE_API_KEY}`,
+    basePath: process.env.CONNECT_PROFILE_URL,
+  }),
+);
+
+adminClient
+  .deleteProfile(process.env.CONNECT_TEST_ACCOUNT_SUB || "")
+  .then((response) => {
+    if (response.status === HttpStatus.NO_CONTENT) {
+      console.log(
+        `✅ Profile with sub '${process.env.CONNECT_TEST_ACCOUNT_SUB}' deleted from Connect.Profile`,
+      );
+    }
+  })
+  .catch((error) => {
+    if (!(error.message === "Request failed with status code 404")) {
+      console.error("💥", error.message);
+      process.exit(1);
+    }
```

Comment:
The user may not exist, mainly for two reasons:
- we are testing against the mock server
- It has already been deleted
When it is the case, we don't want the script to fail.

Reply:
We already had this script (https://github.com/fewlinesco/connect-account/blob/main/bin/e2e/cleaning-profile-staging.ts), which is used in `config.yml` (l70)

Reply:
Yes, but since it's tailored for staging I preferred to add it only for e2e.
Do you prefer me to tweak this and use the script I wrote in the code you shared to avoid repetition?

Reply:
It isn't really tied to staging, there's just the API_KEY env var to change and it could be used anywhere I think. The script name could also be changed, to not reference staging. Anyway, do as you want, but we don't need to have the both imo.

Reply:
I changed the code:
- removed my function
- tweaked a bit the existing one to be less specific to staging (mainly naming).

I also changed the env variable name and added it to CircleCi (so it is there 2 time for now with two different names but I'll delete the `CONNECT_PROFILE_STAGING_API_KEY` one just before the merge of this PR).